easybuild/tools/module_generator.py

```diff
@@ -1167,6 +1177,19 @@ def __init__(self, *args, **kwargs):
             if self.modules_tool.version and LooseVersion(self.modules_tool.version) >= LooseVersion('7.7.38'):
                 self.DOT_MODULERC = '.modulerc.lua'
 
+    @staticmethod
+    def _path_join_cmd(path):
+        "Return 'pathJoin' command for given path string"
+        path_components = [quote_str(p) for p in path.split(os.path.sep) if p]
+
+        path_root = quote_str(os.path.sep) if os.path.isabs(path) else 'root'
+        path_components.insert(0, path_root)
+
+        if len(path_components) > 1:
+            return f'pathJoin({", ".join(path_components)})'
```

Comment:
Using an f-string here makes this very hard to read, how about

```suggestion
            return 'pathJoin(' + ', '.join(path_components) + ')'

Reply:
fixed in https://github.com/easybuilders/easybuild-framework/commit/510e8f0942598ec8856ffb5d7fefd0c56fac235a